### PR TITLE
Online DDL: vtctl OnlineDDL supports 'complete' command

### PR DESCRIPTION
**Pull request by shlomi-noach**
## Description

Followup to https://github.com/vitessio/vitess/pull/9171
In https://github.com/vitessio/vitess/pull/9171 we introduced `ALTER VITESS_MIGRATION '...' COMPLETE` command. This PR complements the SQL interface by adding a `vtctl OnlineDDL <keyspace> complete <uuid>` command.

To be documented.

## Related Issue(s)

Tracking: #6926

## Checklist
- [ ] Should this PR be backported?
- [ ] Tests were added or are not required
- [ ] Documentation was added or is not required

### CODE REVIEW

File: doc/releasenotes/13_0_0_summary.md [hunk 1]
```diff
@@ -47,6 +47,19 @@ This command indicates that a migration executed with `-postpone-completion` is
...
+
+Complementing the `alter vitess_migration ... complete` query, a migration can also be completed via `vtctl`:
+
+```shell
+vtctl OnlineDDL <keyspace> complete <uuid>
+```
+For example:
+
+```shell
+vtctlclient OnlineDDL commerce complete d08ffe6b_51c9_11ec_9cf2_0a43f95f28a3
```
Comment (deepthi) [inline doc/releasenotes/13_0_0_summary.md:60]: The command template uses `vtctl` and the example uses `vtctlclient`. Better to keep it uniform and use the same (preferably `vtctlclient`) in both places.

File: go/vt/vtctl/vtctl.go [hunk 2]
```diff
@@ -3328,6 +3328,14 @@ func commandOnlineDDL(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag
...
 			query, bindErr = sqlparser.ParseAndBind(`update _vt.schema_migrations set migration_status='retry' where migration_uuid=%a`, sqltypes.StringBindVariable(arg))
 		}
+	case "complete":
+		{
+			if arg == "" {
+				return fmt.Errorf("UUID required")
+			}
+			uuid = arg
+			query, bindErr = sqlparser.ParseAndBind(`update _vt.schema_migrations set migration_status='complete' where migration_uuid=%a`, sqltypes.StringBindVariable(arg))
+		}
```
Comment (deepthi) [inline go/vt/vtctl/vtctl.go:3338]: I haven't seen this style before. Not objecting to it, but the curly braces around the code block aren't really required, right?

**Review by deepthi — APPROVED** [review]
Suggestion on release notes, rest LGTM

File: doc/releasenotes/13_0_0_summary.md (hunk 1 above)
Comment (shlomi-noach) [reply doc/releasenotes/13_0_0_summary.md:60]: fixed

File: go/vt/vtctl/vtctl.go (hunk 2 above)
Comment (shlomi-noach) [reply go/vt/vtctl/vtctl.go]: Right -- these are not needed. To me it reads clearer with the curly braces, though I realize the convention is to not use them. I've just removed them and committed the braceless case statement.

**Pull request merged**